src/components/transport_manager/src/transport_manager_impl.cc

```diff
@@ -426,11 +426,13 @@ int TransportManagerImpl::SearchDevices() {
   return transport_adapter_search;
 }
 
-int TransportManagerImpl::Init() {
-  LOG4CXX_TRACE(logger_, "enter");
-  is_initialized_ = true;
-  LOG4CXX_TRACE(logger_, "exit with E_SUCCESS");
-  return E_SUCCESS;
+int TransportManagerImpl::Init(resumption::LastState &last_state) {
+    // Last state requred to initialize Transport adapters
+    UNUSED(last_state);
```

Comment:
@AByzhynar , TransportManagerDefault inherited from TMI, and in TMD::Init(LastState) TransportAdapters will be inited  with this last state.